packages/fx-core/resource/package.nls.json

```diff
@@ -99,6 +99,22 @@
   "plugins.frontend.checkSystemTimeTip": "You may get expired credentials, check if your system time is correct.",
   "plugins.frontend.restoreEnvironmentTip": "If you manually updated configuration files (under directory .%s), recover them.",
   "plugins.frontend.baseErrorMessage": "%s Suggestions: %s",
+  "plugins.bot.questionHostTypeTrigger.title": "Choose triggers",
+  "plugins.bot.questionHostTypeTrigger.placeholder": "Choose triggers",
+  "plugins.bot.questionHostTypeTrigger.error.emptySelection": "You have to select at least one trigger.",
+  "plugins.bot.questionHostTypeTrigger.error.hostTypeConflict": "Azure Functions and App Service host types are in conflict",
+  "plugins.bot.triggers.functionsTimer.label": "Timer Trigger",
+  "plugins.bot.triggers.functionsTimer.cliName": "timer",
+  "plugins.bot.triggers.functionsTimer.description": "Timer Trigger",
```

Comment:
Give a more meaningful description, you can refer to VS's strings:
https://docs.microsoft.com/en-us/azure/includes/media/functions-vs-tools-create/functions-project-settings.png

Reply:
Updated